### DISCUSSION TITLE
Hacs - error dopo aggiornamento 1.17 (credo)

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#1110] [HUB: Home Assistant]
Dopo riavvio HA, trovo Hacs che è fuori uso, ecco l'errore, sinceramente va oltre le mie capacità, qualcuno sa darmi una mano? grazie

![Cattura](https://user-images.githubusercontent.com/84615011/143538761-46d95b6f-bec4-47d9-b3ff-0c510271ec18.JPG)

**Comment by Scialla** [comment]
Io non ci capisco una fava ma, sembra, si sia incastrato l'update...

proverei a eliminare l'integrazione e reinstallarla nuovamente

**Reply by calz64** [reply]
Ho dato un'occhiata a come fare a reinstallare Hacs e devo dire che non ricordavo. Devo rifare tutti i passaggi del token etc.???
Ma se reinstallo perdo tutte le integrazioni che ho installato? 
Però queste integrazioni funzionano anche se Hacs non è raggiungibile, come è possibile?
Forse Hacs è solo l'interfaccia per la 'gestione'? 
ho un po' di confusione... grazie comunque cercherò di capire meglio.





'

**Reply by snakuzzo** [reply]
Sul sito di HACS c'è scritto...

_Home Assistant Community Store
HACS gives you a powerful UI to handle downloads of all your custom needs._

Ti espone semplicemente una webgui per poter installare i custom component/custom card (o almeno quelli che possono essere installati tramite HACS).

Ciò non toglie che tutti i custom puoi installarli anche senza usare HACS e tali custom funzionano anche senza HACS

Resta il fatto che se non sei pratico, HACS ti torna utile e non vedo alcun problema a reinstallarlo seguendo la procedura che, permettimi di dire, è più che banale (e non c'entra l'eventuale esperienza che uno possa avere)

**Comment by calz64** [comment]
Ok Diego grazie della fiducia ma non è detto: comunque basta cancellare fisicamente la directory, caricarne una nuova con zip scompattato e riavviare HA e poi seguire configurazione? 
mi puoi confermare, grazie

Pensi anche tu che sia un problema di mancato buon fine di aggiornamento?

**Comment by calz64** [comment]
Problema -  ho dovuto fare downgrade fino alla versione 1.15 perché HACS successivi non sono compatibili col mio core: sono fermo 2021.9.7 per problemi con Tuya e mio allarme.
